02_knight_and_princess/danil_dyachenko/graph_gen.cpp

```diff
@@ -0,0 +1,126 @@
+#include <array>
+#include <fstream>
+#include <iostream>
+#include <vector>
+
+using VertexId = int;
+
+struct Vertex {
+  VertexId id;
+  std::vector<int> edges;
+};
+
+struct Edge {
+  VertexId id;
```

Comment:
Наверное, это `EdgeId`, а не `VertexId` :)))
```suggestion
  EdgeId id;
```